site/src/util/mdx-utils.tsx

```diff
@@ -68,13 +71,13 @@ const getHeadingsFromParent = (parent: Parent): Heading[] =>
           type: "heading",
           /** @todo: don't assume that FAQ accordions are always headings at depth 3 */
           depth: 3,
+          anchor: child.attributes.find(({ name }) => name === "anchor")?.value,
```

Comment:
These don't exist any more

Reply:
Removed in 90b9a7c2d74f6b9a3b894ae5913e9a3585295dba